server.go

```diff
@@ -554,6 +571,27 @@ func NewFromConfig(logger *logrus.Logger, conf Config) (*Server, error) {
 			}
 		}
 
+		if conf.NewrelicInsertKey != "" {
+			nrSpanSink, err := newrelic.NewNewRelicSpanSink(
```

Comment:
Unless I'm missing something, won't this (combined with the above) mean that two separate copies of the New Relic sink will be created and configured if Datadog is also configured?

Reply:
The diff looks weird, but this block is nested under:

```
	// Configure tracing sinks
	if len(conf.SsfListenAddresses) > 0 {
```

Not the Datadog section.

Two different New Relic sinks (`NewNewRelicSpanSink` and `NewNewRelicMetricSink` above) will be created, depending on which is enabled via config.  I agree this is not the ideal state, but follows the method used by others.

Reply:
@ChimeraCoder is there something else I can do to clarify this concern and move the PR forward?

Reply:
Nope, that's it - thanks for the explanation!